### DISCUSSION TITLE
rate-limit lockout for ratelimit-ip-login-076ba4a461bd1d5a in login scope from xxx.xxx.xxx.xxx

### DISCUSSION THREAD

**Discussion by ecxod** [WeblateOrg/weblate#8909] [General]
How can I turn off this rate-limit thing?

**Comment by nijel** [comment]
Search for rate limit in the documentation :⁠-⁠)

**Comment by ecxod** [comment]
Thank you. very helpfull.

just for your info I upgraded to the last version with pip 4.16.2 (or 4.1.7 who knows), now i cannot connect any more.


Method used :
```
# stop celery
(weblate-env) weblate@obsi:~$ ~/WL/lib/python3.9/site-packages/weblate/examples/celery stop

# as root stop celery
root@obsi:~# systemctl stop celery-weblate.service

# check the latest version out
(weblate-env) weblate@obsi:~$ pip install -U "Weblate[all]"

# start celery
(weblate-env) weblate@obsi:~$ ~/WL/lib/python3.9/site-packages/weblate/examples/celery start

# Finally as root
root@obsi:~# systemctl start celery-weblate.service
root@obsi:~# systemctl status celery-weblate.service

? celery-weblate.service - Celery Service (Weblate)
     Loaded: loaded (/lib/systemd/system/celery-weblate.service; bad; preset: enabled)
     Active: inactive (dead)
```
The system is totally broken, but if I check it it says

```
$ weblate check
System check identified no issues (1 silenced).
```


and bthe way
$ weblate check --version
4.1.7
Webalte version
4.1.7

when I was on 4.15.2 it showed 4.0.6

have a nice evening

**Reply by nijel** [reply]
You apparently have more versions installed in different ways. Starting by example celery script is not same as using the systemd service, you should choose just one of these.
